msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md

```diff
@@ -1046,7 +1046,7 @@ The Microsoft 365 App Compliance Program is intended to help organizations asses
 
 * **Publisher Verification**: Helps admins and end users understand the authenticity of app developers integrating with the Microsoft identity platform. When completed, a blue **verified** badge displays on the Azure Active Directory consent dialog and other screens. For more information, see [Mark your app as publisher verified](/azure/active-directory/develop/mark-app-as-publisher-verified).
 
-    :::image type="content" source="../../../../assets/images/submission/validation-365-compliance-publisher-verification.png" alt-text="validation-365-compliance-publisher-verifications":::
+    :::image type="content" source="../../../../assets/images/submission/validation-365-compliance-publisher-verification.png" alt-text="The screenshot shows the validation 365 compliance publisher verifications.":::
```

Comment:
What does this mean? `the validation 365 compliance publisher verifications`

Reply:
FYI, @v-habibaumme

Reply:
resolved

Reply:
Needs re-writing to show exactly what the screenshot means to show - it is not just validation. Please get it peer reviewed internally.

Reply:
resolved